### PR TITLE
setup-cml and setup-dvc with cml-runner update

### PR DESCRIPTION
**Pull request by DavidGOrtega**
Update to be using setup-cml with cml-runner

### CODE REVIEW

**Review by 0x2b3bfa0 — APPROVED** [review]
Argh! Q3 2022 using `docker-machine`

File: .github/workflows/cml.yaml [hunk 1]
```diff
@@ -3,58 +3,48 @@ name: style-transfer
...
+    container:
+      image: dvcorg/cml-py3
+      options: --gpus all
     
     steps:
       - uses: actions/checkout@v2
 
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
```
Comment (0x2b3bfa0) [inline .github/workflows/cml.yaml:39]: It's more like 16, right?

File: .github/workflows/cml.yaml [hunk 2]
```diff
@@ -3,58 +3,48 @@ name: style-transfer
...
+            --cloud-spot \
+            --labels=cml-runner-gpu
+          
   train:
-    needs: deploy-cloud-runner
-    runs-on: [self-hosted,cml]
+    needs: deploy-runner
+    runs-on: [self-hosted,cml-runner-gpu]
+    container:
+      image: dvcorg/cml-py3
```
Comment (0x2b3bfa0) [inline .github/workflows/cml.yaml:31]: `-Wdeprecated-container-image`

**Review by 0x2b3bfa0 — CHANGES_REQUESTED** [review]
See above

**Comment by DavidGOrtega** [comment]
Not relevant any more

**Pull request closed without merging**